tips/TIP-0048/openapi3-core.yaml

```
@@ -0,0 +1,3376 @@
+        Check the readiness of the node to issue a new block, the reference mana cost based on the rate setter and current network congestion, and the block issuance credits of the requested account.
+      parameters:
+        - in: path
+          name: accountId
+          schema:
+            type: string
+          example: "0xf532a53545103276b46876c473846d98648ee418468bce76df4868648dd73e5d"
+          required: true
+          description: Identifier of the account.
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CongestionResponse'
+              examples:
+                CongestionResponse:
+                  $ref: '#/components/examples/get-congestion-estimate-response-example'
+        '400':
+          description: "Unsuccessful operation: indicates that the provided data is invalid."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequestResponse'
+        '403':
+          description: "Unsuccessful operation: indicates that the endpoint is not available for public use."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ForbiddenResponse'
+        '500':
+          description: "Unsuccessful operation: indicates that an unexpected, internal server error happened which prevented the node from fulfilling the request."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InternalErrorResponse'
+        '503':
+          description: "Unsuccessful operation: indicates that the node is not synced."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceUnavailableResponse'
+
+  '/api/core/v3/rewards/{outputId}':
+    get:
+      tags:
+        - rewards
+      summary: Returns the totally available Mana rewards of an account or delegation output decayed up to endEpoch index provided in the response.
+      description: >-
+        Returns the totally available Mana rewards of an account or delegation output up until the current epoch index. 
+        Note that rewards for an epoch only become available at the beginning of the next epoch. 
+        If the end epoch of a staking feature is equal or greater than the current epoch, the rewards response will not include the potential future rewards for those epochs.
+        startEpoch and endEpoch indicates the actual range for which reward value is returned and decayed for.
+      parameters:
+        - in: path
+          name: outputId
+          schema:
+            type: string
+          example: "0xf532a53545103276b46876c473846d98648ee418468bce76df4868648dd73e5d"
+          required: true
+          description: Identifier of the account or delegation output.
```

```suggestion
          name: outputId
          schema:
            type: string
          example: "0xf532a53545103276b46876c473846d98648ee418468bce76df4868648dd73e5d"
          required: true
          description: Output ID of an account or delegation output.
```

We take an output ID here and the way it's written currently could be understood as "this takes an account or delegation ID".